.github/workflows/acceptance-tests.yaml

```diff
@@ -59,12 +62,15 @@ jobs:
           # This is an API key generated as the Marvin user (marvin@prefect.io).
           # We do this because the User API requires requests to come from an API
           # key generated by that same user.
+          # The value is stored in 1Password: `Terraform provider acceptance test secrets`.
           PREFECT_API_KEY: ${{ secrets.ACC_TEST_PREFECT_MARVIN_API_KEY }}
           PREFECT_CLOUD_ACCOUNT_ID: ${{ secrets.ACC_TEST_PREFECT_CLOUD_ACCOUNT_ID }}
-          ACC_TEST_USER_RESOURCE: "yes"
           # This is the user ID retrieved by logging in as the Marvin user
           # and copying the ID on the "Profile" page.
           ACC_TEST_USER_RESOURCE_ID: "48750018-b4c4-4484-8fbf-b61baf3926b5"
           # This ensures only the User resource tests run, instead of running
           # all the tests again because that was completed in the last step.
-          TESTS: "TestAccResource_user"
+          # Regex ensures that the following tests are included:
+          # - TestAccResource_user
+          # - TestAccResourcE_user_api_key
+          TESTS: "TestAccResource_user*"
```

Comment:
good catch - i thought it was already doing this pattern match, but i guess it was running all the tests